### PR TITLE
Add an all in one command which combines apiserver and controller-manager

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: cmd/allinone/app/entrypoint.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+	// Load configuration from file
+	conf, err := config.TryLoadFromDisk()
+	if err == nil {
+		s = &options.ServerRunOptions{
+			GenericServerRunOptions: s.GenericServerRunOptions,
+			Config:                  conf,
+		}
+	} else {
+		klog.Fatal("Failed to load configuration from disk", err)
```
Comment (JohnNiang) [inline cmd/allinone/app/entrypoint.go:43]: It's better to log as warning or break current thread.

File: cmd/allinone/app/entrypoint.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		s = &controllerOpt.DevOpsControllerManagerOptions{
+			KubernetesOptions: conf.KubernetesOptions,
+			JenkinsOptions:    conf.DevopsOptions,
+			S3Options:         conf.S3Options,
+			LeaderElection:    s.LeaderElection,
+			LeaderElect:       s.LeaderElect,
+			WebhookCertDir:    s.WebhookCertDir,
+		}
+	} else {
+		klog.Fatal("Failed to load configuration from disk", err)
```
Comment (JohnNiang) [inline cmd/allinone/app/entrypoint.go:68]: See above.

File: cmd/allinone/main.go [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+import (
+	"kubesphere.io/devops/cmd/allinone/app"
+	"log"
+)
+
+func main() {
+	cmd := app.NewCommand()
+	if err := cmd.Execute(); err != nil {
+		log.Fatalln(err)
```
Comment (JohnNiang) [inline cmd/allinone/main.go:11]: Need append `os.Exit(1)` below here.

**Review by JohnNiang — CHANGES_REQUESTED** [review]

File: cmd/allinone/app/entrypoint.go (hunk 1 above)
Comment (LinuxSuRen) [reply cmd/allinone/app/entrypoint.go:43]: Actually, it's not true. It cannot continue without a configuration.

File: cmd/allinone/main.go (hunk 3 above)
Comment (LinuxSuRen) [reply cmd/allinone/main.go:11]: It's ok if it returns an error code once the command line exit with an error.

image

**Pull request merged**